manifests/init.pp

```diff
@@ -117,7 +125,12 @@
   $thumbslug_oauth_key = $candlepin::params::thumbslug_oauth_key,
   $thumbslug_oauth_secret = $candlepin::params::thumbslug_oauth_secret,
 
+  $keystore_file = $candlepin::params::keystore_file,
   $keystore_password = $candlepin::params::keystore_password,
+  $keystore_type = $candlepin::params::keystore_type,
+  $truststore_file = $candlepin::params::truststore_file,
+  $truststore_password = $candlepin::params::truststore_password,
```

Comment:
Should we validate that the file is an absolute path or present? Should we validate that a password is passed in and is defined/a string?